src/transactions/saga.ts

```diff
@@ -225,7 +234,60 @@ function* watchAddressToE164PhoneNumberUpdate() {
   )
 }
 
+function* getTransactionReceipt(
```

Comment:
could we add a test for this? 🙏🏻

Reply:
I added some test for `watchPendingTransactions` which covers this method. Let me know if that's enough :)